front/src/components/UserForm.tsx

```diff
@@ -155,7 +155,7 @@ const UserFormDesign: React.FC = () => {
                                           <TextField
                                                  label="Name"
                                                  {...register("username", {
```

Comment:
The most importnat part for the update is to set the `register()` value
it must be ` {...register("name", {`
all lower case